tests/components/hvv_departures/patched_data.py

```diff
@@ -0,0 +1,194 @@
+"""Patched data for tests."""
```

Comment:
Can we store these fixtures as json files instead under `/tests/fixtures/hvv_departures/`? We have a test helper to load fixtures in `tests/common.py`.